apps/dashboard/src/modules/attendance/components/PoolForm/utils.tsx

```diff
@@ -0,0 +1,21 @@
+// validates and returns {max, min } if valid
+export function validateAndReturn(values: number[]): number[] {
```

Comment:
Suggest to make the name more descriptive

```suggestion
export function clampStudyYears(values: number[]): number[] {
```